File: Week 01/id_343/LeetCode_26.go
func removeDuplicates(nums []int) int {
    if len(nums) <= 1 {
        return len(nums)
    }
    
    lens, start := 1, nums[0]
    for i := 1; i < len(nums); i++ {
        if nums[i] != start {
            lens++
            nums[lens-1], start = nums[i], nums[i]
        }
    }
    return lens
}

File: Week 01/id_113/LeetCode26.py
class Solution:
    def removeDuplicates(self, nums: list[int]) -> int:
        p, q = 0, 1
        while q < len(nums):
            if nums[p] != nums[q]:
                nums[p + 1] = nums[q]
                p += 1
            q += 1

        return p + 1
